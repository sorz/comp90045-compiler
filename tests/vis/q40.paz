program q40;

var n: real;

procedure f(x: real);
begin
    x := x + 1;
    write(x);
    writeln
end;

begin
    n := 41;
    f(n);
    write(n);
    writeln
end.
