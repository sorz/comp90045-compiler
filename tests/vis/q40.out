42.000000
41.000000
